fix: use different pagination types for the queries of a resource (#4453)
api-platform/core: CHANGELOG.md

## 2.7.0

> * GraphQL: Add ability to use different pagination type from queries of a same resource (#4453)

Reviewer: ```suggestion
* GraphQL: Add ability to use different pagination types for the queries of a resource (#4453)
```
